src/components/Pagination/Pagination.tsx

```diff
@@ -144,6 +144,7 @@ export function Pagination({
       nextButton
     );
 
+  const previousHandler = onPrevious || noop;
```

Comment:
What do you think to setting noop as the default value when we destructure the arguments of the Pagination function?

```
onNext = noop,
onPrevious = noop,
```

Then the handleCallback call would continue to be `handleCallback(onPrevious)`.

We do that in a bunch of other places, e.g. ChoiceList onChange (search for `= noop`)

Reply:
Unfortunately, that'll break some conditional logic we have for deciding whether to or not to render a listener `(previousURL || onPrevious) &&`